src/bin/rgb.rs

```diff
@@ -382,11 +387,43 @@ fn main() -> Result<(), Error> {
             }
         },
         Command::Psbt { subcommand } => match subcommand {
-            PsbtCommand::Bundle { psbt_in, psbt_out } => {
+            PsbtCommand::Bundle {
+                psbt_in,
+                psbt_out,
+                method,
+            } => {
                 let psbt_bytes = fs::read(&psbt_in)?;
                 let mut psbt = Psbt::deserialize(&psbt_bytes)?;
 
-                let count = psbt.rgb_bundle_to_lnpbp4()?;
+                let mut count: usize = 0;
+                match method {
+                    CloseMethod::TapretFirst => {
+                        if let Some(output) =
+                            psbt.outputs.iter_mut().find(|o| o.script.is_v1_p2tr())
+                        {
+                            if output.tapret_dfs_path().is_none() {
+                                output
+                                    .set_tapret_dfs_path(&DfsPath::with([&DfsOrder::First]))
+                                    .expect("given output should be valid");
```

Comment:
This method errors not when the output doesn't exist (since we have an output object, so it indeed exists) but when another commitment is present. So we must not `expect` here but instead perform a proper merge operation. We may leave it for a follow-up PR tough, but it this case we should do `?` and propagate the error.

From the docs on the method:
```rust
/// Errors with [`TapretKeyError::OutputAlreadyHasCommitment`] if the
/// commitment is already present in the output.
```

Reply:
now propagating the error